sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -391,10 +391,10 @@ class AnalyzeSentimentResult(DictMixin):
         field will contain information about the document payload.
     :type statistics:
         ~azure.ai.textanalytics.TextDocumentStatistics
-    :param sentiment_scores: Document level sentiment confidence
+    :param confidence_scores: Document level sentiment confidence
```

Comment:
should we change `class` for `label`?
Also, could you add to docstrings that the score represents the Softmax score.

Reply:
>Also, could you add to docstrings that the score represents the Softmax score.

NV. just saw it down

Reply:
Changed to `label`.